src/TropicalGeometry/hypersurface.jl

```diff
@@ -248,9 +248,9 @@ end
 
 Return the tropical polynomial used to construct `TropH`.  Raises an error if it is not cached.
 """
-function tropical_polynomial(TropH::TropicalHypersurface)
+function tropical_polynomial(TropH::TropicalHypersurface{minOrMax}) where minOrMax
```

Comment:
```suggestion
function tropical_polynomial(TropH::TropicalHypersurface{minOrMax,true}) where minOrMax
```

`TropicalHypersurface` has two parameters.

Reply:
The second parameter is always `true` (inherited from `TropicalVarietySupertype`).

Reply:
But one can omit trailing type parameters.

Reply:
(and if they are not used in a given context, it is usually a good idea to omit them, too)

Reply:
Ah, I understand now. Thanks for the clarification!